main.go

```diff
@@ -10,8 +10,27 @@ import (
 
 	"github.com/shawn-hurley/jsonrpc-golang/jsonrpc2"
 	"github.com/shawn-hurley/jsonrpc-golang/lsp/protocol"
+	"github.com/shawn-hurley/jsonrpc-golang/rules"
 )
 
+//TODO(shawn-hurley) - this needs to be passed in as args. Will need to refactor to use cobra
+var config = rules.Configuration{
+	ProjectLocation: "./examples/golang",
+}
+
+//TODO(shawn-hurley) - this package/type name stutters.
+var workspaceRules = []rules.Rule{
+	{
+		ImportRule: &rules.ImportRule{
+			GoImportRule: &rules.GoImportRule{
+				Import: "pkg/apis/apiextensions/v1beta1.CustomResourceDefinition",
```

Comment:
Would we want to separate this into import/reference, ie 
```
import: k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1
reference: CustomResourceDefinition
```
so as to avoid requiring the user to know the syntax for the query?

Reply:
Lets change, this in a follow up along with the others